<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCompany</class>
 <widget class="QWidget" name="AddCompany">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddCompany</string>
  </property>
  <widget class="QGroupBox" name="add_company_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>561</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{background:url(:/Comcalen/background.jpg)}</string>
   </property>
   <property name="title">
    <string>Add Company</string>
   </property>
   <widget class="QLineEdit" name="cname">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="line_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Company name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="employer_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="empname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Your name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="employer_surname">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="empsurname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Your surname:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_employee_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add employee</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_company">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Company</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_main_w">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="adding_employee">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{background:url(:/Comcalen/background.jpg)}</string>
   </property>
   <property name="title">
    <string>Add Employee</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Employee Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="employee_name">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>92</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Employee Surname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="employee_surname">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="employee_salary">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Salary</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="employee_hl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hours limit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_employee">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add employee</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Comcalen.qrc"/>
 </resources>
 <connections/>
</ui>
